### PR TITLE
Calico integration with mesos 0.26.0 & marathon 0.14

### PR DESCRIPTION
**Pull request by Zogg**
## DO NOT MERGE UNTIL MESOS 0.26 OR LATER IS USED IN MANTL

Before merge:
 * update bintray's repositories and mesos packaging names

Features:
* calico is packaged into rpm
* mesos is configured to use calico
* mesos 0.26 & marathon 0.14 used

### CODE REVIEW

File: roles/marathon/defaults/main.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 marathon_version: 0.13.0
-marathon_package: "https://bintray.com/artifact/download/asteris/mantl-rpm/marathon-{{ marathon_version }}-1.x86_64.rpm"
+marathon_package: "https://bintray.com/artifact/download/asteris/testing/marathon-0.14.0_RC2-1.x86_64.rpm"
```
Comment (djosborne) [inline roles/marathon/defaults/main.yml]: Marathon cut an official 0.14.0 release, so no need to use RC2 anymore

**Comment by mcapuccini** [comment]
@stevendborrelli so If I want to use Calico, should I checkout this branch? I tried to add Calico roles on MANTL 1.0.2 and it doesn't seem to work.

**Comment by Zogg** [comment]
@mcapuccini If you want the latest and greatest of calico/mesos/kube, use this branch: https://github.com/CiscoCloud/mantl/pull/1133 .

**Comment by siddharthist** [comment]
@Zogg What's the status of this PR? Is it relevant, or will we use the new Mesos/Calico/Marathon packages before this goes in?

**Comment by stevendborrelli** [comment]
Closing this in preference of newer PRs for mesos/calico.

**Pull request closed without merging**